THEMES.md

```diff
@@ -2,6 +2,24 @@
 
 All available themes are only best effort ports by myself/ other users. If you find a theme to be weird/ wrong please open an issue/ pr.
 
+### ayu dark
```

Comment:
all names should be changed to work directly in lualine. 
```vim
:lua require('lualine').theme = 'ayu dark'
```
Obviously breaks, should be `ayu_dark` in THEMES.md

Reply:
fixed